source/draw/RasterYuv420Semiplanar.ooc

```diff
@@ -203,7 +236,7 @@ RasterYuv420Semiplanar: class extends RasterYuvSemiplanar {
 		(RasterMonochrome new(buffer slice(0, yLength), size, stride), RasterUv new(buffer slice(uvOffset, uvLength), This _uvSize(size), stride))
 	}
 	_uvSize: static func (size: IntVector2D) -> IntVector2D {
-		IntVector2D new(size x / 2 + (size x isOdd ? 1 : 0), size y / 2 + (size y isOdd ? 1 : 0))
+		IntVector2D new((size x + 1) / 2, (size y + 1) / 2)
```

Comment:
Thanks @sebastianbaginski for the info.
according to the C99 standard, the expression (a/b)*b + a%b shall equal a, so if (5/2) == 3, then 3*2 + 1 != 5. It must be 2 according to the C99 : [http://port70.net/~nsz/c/c99/n1256.html#6.5.5p6](url)

@davidpiuva, i'm fine with this one now.